transmart-core-db-tests/src/integration-test/groovy/org/transmartproject/db/dataquery2/HypercubeIntegrationSpec.groovy

```diff
@@ -36,11 +36,8 @@ class HypercubeIntegrationSpec extends TransmartSpecification {
         setupData()
 
         def hypercube = queryResource.doQuery(constraints: [study: [clinicalData.longitudinalStudy.studyId]])
-        def resultObs = Lists.newArrayList(hypercube)
+        def resultObs = Lists.newArrayList(hypercube).sort()
```

Comment:
@gijskant Could you at least review these kinds of things? HypercubeValues are not comparable.